Xamarin.Forms.Core/BindableLayout.cs

```diff
@@ -0,0 +1,261 @@
+﻿using System.Collections;
+using System.Collections.Specialized;
+
+namespace Xamarin.Forms.Core
+{
+	public static class BindableLayout
+	{
+		public static readonly BindableProperty ItemsSourceProperty =
+			BindableProperty.Create("ItemsSource", typeof(IList), typeof(Layout<View>), default(IList),
+				propertyChanged: (b, o, n) => { GetController(b).ItemsSource = (IList)n; });
+
+		public static readonly BindableProperty ItemTemplateProperty =
+			BindableProperty.Create("ItemTemplate", typeof(DataTemplate), typeof(Layout<View>),
+				propertyChanged: (b, o, n) => { GetController(b).ItemTemplate = (DataTemplate)n; });
+
+		public static readonly BindableProperty ItemTemplateSelectorProperty =
+			BindableProperty.Create("ItemTemplateSelector", typeof(DataTemplateSelector), typeof(Layout<View>),
+				propertyChanged: (b, o, n) => { GetController(b).ItemTemplateSelector = (DataTemplateSelector)n; });
+
+		public static readonly BindableProperty ControllerProperty =
```

Comment:
shouldn't be `public`

Reply:
If it's not public it completely eliminates the possibility to override item view setup. 
I tried to describe in detail in the comments in the code example above the reasoning behind controller. Maybe I wasn't clear enough? 
I'd like to hear more thoughts. Thanks!

Reply:
Should I maybe provide an example to show how overriding the controller makes sense?

Reply:
The following two comments you made are related to this too.

Reply:
Or maybe I make it private for now and open a separate enhancement PR to make it public? 
Would this be quicker in order to get this functionality merged and released?

Reply:
Done, I removed public. I think it's a missed opportunity